admin.py

```diff
@@ -107,21 +109,54 @@ class TicketInline(admin.TabularInline):
     extra = 0
 
 
+def process_names_csv(csv_file):
+    """
+    Process a csv file for online orders
+    @param csv_file: A csv file containing names to be set to paid.
+    @return: list of names that had problems being processed and a message summarizing the done process.
+    """
+    names = pandas.read_csv(csv_file, delimiter="\t", header=None)
```

Comment:
Since you use pandas as a very basic csv reader, I would prefer to just use the default csv reader.

import csv

with open(csv_file) as f:
    csv_reader = csv.reader(f, delimiter=',')
    for index, data in enumerate(csv_reader):
           pass

It saves on additional requirement and the code is almost the same...

Reply:
done :)